db/system_keyspace.cc

```diff
@@ -603,6 +613,7 @@ schema_ptr system_keyspace::large_rows() {
                 .with_version(generate_schema_version(id))
                 .set_gc_grace_seconds(0)
                 .set_caching_options(caching_options::get_disabled_caching_options())
+                .set_wait_for_sync_to_commitlog(true)
```

Comment:
This table is certainly not critical, it is informative only (the database itself never reads it).

Reply:
I guess it's the same with large_partitions?